### PR TITLE
posix : implement sched_set/getscheduler and sched_set/getparam

### PR DESCRIPTION
**Pull request by zetwhite**
move #3089 implements to posix.rs

### CODE REVIEW

File: vm/src/stdlib/posix.rs [hunk 1]
```diff
@@ -39,8 +39,13 @@ pub mod module {
             _os, fs_metadata,
         },
         utils::{Either, ToCString},
-        IntoPyObject, ItemProtocol, PyObjectRef, PyResult, PyValue, StaticType, TryFromObject,
+<<<<<<< HEAD
```
Comment (youknowone) [inline vm/src/stdlib/posix.rs:42]: merge is not fully finished

File: vm/src/stdlib/posix.rs [hunk 2]
```diff
@@ -39,8 +39,8 @@ pub mod module {
             _os, fs_metadata,
         },
         utils::{Either, ToCString},
-        IntoPyObject, ItemProtocol, PyObjectRef, PyResult, PyValue, StaticType, TryFromObject,
-        VirtualMachine,
+        IntoPyObject, ItemProtocol, PyObjectRef, PyRef, PyResult, PyValue, StaticType,
```
Comment (youknowone) [inline vm/src/stdlib/posix.rs:42]: @zetwhite could you check these unused import warnings?

File: vm/src/stdlib/posix.rs [hunk 3]
```diff
@@ -473,12 +473,138 @@ pub mod module {
...
+        target_os = "android"
+    ))]
+    #[derive(FromArgs)]
+    struct SchedSetschedulerArgs {
+        #[pyarg(positional)]
+        pid: i32,
+        #[pyarg(positional)]
+        policy: i32,
+        #[pyarg(positional)]
+        sched_param_obj: PyRef<SchedParam>,
```
Comment (youknowone) [inline vm/src/stdlib/posix.rs:541]:
```suggestion
        sched_param_obj: crate::PyRef<SchedParam>,
```

will remove the warning

File: vm/src/stdlib/posix.rs [hunk 4]
```diff
@@ -473,12 +473,138 @@ pub mod module {
...
             ))
         }
+
+        #[cfg(any(
+            target_os = "linux",
+            target_os = "netbsd",
+            target_os = "freebsd",
+            target_os = "android"
+        ))]
+        fn try_to_libc(&self, vm: &VirtualMachine) -> PyResult<libc::sched_param> {
```
Comment (youknowone) [inline vm/src/stdlib/posix.rs:483]:
```rust
use crate::TypeProtocol;
```
here

**Review by youknowone — CHANGES_REQUESTED** [review]

File: vm/src/stdlib/posix.rs (hunk 2 above)
Comment (zetwhite) [reply vm/src/stdlib/posix.rs:42]: Ok. i am going to check it

**Review by youknowone — APPROVED** [review]

**Pull request merged**